posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -0,0 +1,353 @@
+---
+layout: post
+title: "Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos and a number of enhancements in Open Liberty 21.0.0.4"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos and a number of enhancements in Open Liberty 21.0.0.4 - OpenLiberty.io
+seo-description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos authentication and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
+blog_description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos authentication and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
```

Comment:
Change the two descriptions to:
Admin Center, a web-based UI to help configure, manage, and monitor your server, and MicroProfile Context Propagation 1.2 are now available in Open Liberty.  Several other enhancements have also been included in this release, including: LDAP connections supporting Kerberos authentication, more control over thread dumps in the Request Timing feature, automatic cleanup of leaked connections, and Single Sign-On (SSO) LTPA and JWT gaining support for authentication filters.

Reply:
Done

Reply:
Changing the descriptions to ```seo-description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos authentication and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)``` caused meta-data issues on draft. This was caused by the `:` in ```...release, including: LDAP connections... ```, I've changed this colon to a semi-colon. Is this okay? @mbroz2